nservicebus/scalability-and-ha/scale-out.md

```diff
@@ -55,6 +55,7 @@ snippet:Routing-StaticRoutes-Endpoint-Msmq
 
 NOTE: System administrators are able to spin-up new instances of the endpoint should the load increase and the only requirement is adding an entry to the routing file. No changes in the source code are required.
 
+WARNING: When using this scaling out technique in a mixed version environment make sure to deploy a distributor in front of the scaled out V6 endpoint if that endpoint needs to subscribe to events published by endpoints using versions lower than 6. Please refer to the V6 version of [the distributor sample](/samples/scaleout/distributor/) for details on how to do it. Otherwise the subscribed events will be delivered to every instance of the scaled out endpoint (instead of being delivered to a single instance using a load-balancing algorithm).
```

Comment:
@SzymonPobiega - I'd remove the word "NOTE" in the above paragraph and keep this one as WARNING.

Rather than add this as Warning, please describe this as 2 use cases.
What happens when:
- a v6 subscriber is scaled out
- a v6 publisher is scaled out. 

and how it interacts with version 5 endpoints.